src/SonarQube.Client/ISSEConnectionFactory.cs

```diff
@@ -0,0 +1,31 @@
+﻿/*
```

Comment:
Copied directly from the prototype PR -> https://github.com/SonarSource/sonarqube-webclient-dotnet/pull/161